settings.gradle

```diff
@@ -20,6 +20,18 @@ plugins {
     id 'com.gradle.enterprise' version '3.11.1'
 }
 
+dependencyResolutionManagement {
+    versionCatalogs {
+        junitVariableVersionLibs {
+            from(files('gradle/libs.versions.toml'))
+            def junitVersion = rootProject.hasProperty('junitVersion') ? rootProject.junitVersion : null
```

Comment:
Isn't this the replacement of build.gradle l.60? Why was null chosen here?

Reply:
well, we only overwrite the original if it is not null. But you are right, this can be written more elegantly.